test/autoprefixer.test.js

```diff
@@ -145,9 +145,33 @@ it('throws on wrong options', () => {
     }).toThrowError(/browsers/);
 });
 
-it('sets options', () => {
-    const opts = { browsers: ['chrome 25', 'opera 12'], cascade: false };
-    expect(autoprefixer(opts).options).toEqual(opts);
+describe('sets options via', () => {
```

Comment:
We don’t use this type of grouping in other tests. It will be better to remove `describe` in terms of tests consistency